File: python/exo15.py
def somme() :

    nbr=-1
    somme = 0
    while nbr < 0:
     nbr = int(input(" Entrer un nombre"))
    i =1
    while i  <= nbr :
     somme = somme + i
     i = i+1

    print("La somme de 1 à " + str(nbr)+ " est : " +str(somme))


if __name__ == '__main__':
        somme()

File: tdAlgo/src/exo15.java
import java.util.Scanner;

public class exo15 {

    public static void main(String [] args){
        Scanner scan = new Scanner(System.in);
        int nbr, somme ,i ;


        do {
            System.out.println(" Entrer un nombre");
            nbr = scan.nextInt();
        }while (nbr <0);

        somme  = 0;
        for (i =1 ;i <= nbr ;i++) {
            somme = somme + i;
        }
        System.out.println("La somme de 1 à "+nbr+" est : "+ somme);

    }
}
